data/tausiq2003.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "Tausiq Samantaray",
+  "type": "personal",
+  "bio": "CS Student | Major focus: **Web dev** and **software dev** | Exploring different technologies",
+  "tags": [
+    "Open Source",
+    "Python",
+    "Web Dev",
+    "Java"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/tausiqtweets" },
+    { "icon": "FaGithub", "url": "https://github.com/tausiq2003" },
+    { "icon": "FaLinkedIn", "url": "www.linkedin.com/in/tausiq-samantaray" }
+  ],
+  "avatar": "https://github.com/tausiq2003.png",
```

Comment:
As of our recent update, we no longer need the `avatar` key. So please remove this from your `json`.

```suggestion
```